src/librustdoc/clean/inline.rs

```diff
@@ -62,7 +62,7 @@ pub(crate) fn try_inline(
         Res::Def(DefKind::Trait, did) => {
             record_extern_fqn(cx, did, ItemType::Trait);
             build_impls(cx, Some(parent_module), did, attrs, &mut ret);
-            clean::TraitItem(build_external_trait(cx, did))
+            clean::TraitItem(Box::new(build_external_trait(cx, did)))
```

Comment:
Just below you use the "box syntax". Is there a reason why not here?

Reply:
that's a match statement

Reply:
Oh you're right. Thanks for the explanation!

Reply:
it confused me at first too